### PR TITLE
BUGZ-489: Make sure the flush timer is not spawned too early

### PR DESCRIPTION
**Pull request by Atlante45**
[JIRA ticket](https://highfidelity.atlassian.net/browse/BUGZ-489)

### CODE REVIEW

File: libraries/shared/src/LogHandler.h [hunk 1]
```diff
@@ -54,12 +54,11 @@ class LogHandler : public QObject {
...
     void printRepeatedMessage(int messageID, LogMsgType type, const QMessageLogContext& context, const QString &message);
 
-private slots:
     void setupRepeatedMessageFlusher();
 
 private:
-    LogHandler();
-    ~LogHandler();
+    LogHandler() = default;
+    ~LogHandler() = default;
```
Comment (SimonWalton-HiFi) [inline libraries/shared/src/LogHandler.h]: Any particular reason not to just remove the lines?

Comment (Atlante45) [reply libraries/shared/src/LogHandler.h]: To keep them private

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21399/HighFidelity-Beta-PR16217-527b27b.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21399/HighFidelity-Beta-Interface-PR16217-527b27b.dmg

File: libraries/shared/src/LogHandler.h (hunk 1 above)
Comment (SimonWalton-HiFi) [reply libraries/shared/src/LogHandler.h]: ok

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21399/HighFidelity-Beta-PR16217-527b27b.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21399/HighFidelity-Quest-Beta-PR16217-527b27b.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21399/HighFidelity-Beta-PR16217-527b27b.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21399/HighFidelity-Beta-Interface-PR16217-527b27b.exe

**Pull request merged**